components/locale-provider/injectLocale.tsx

```diff
@@ -8,8 +8,12 @@ export interface ComponentContext {
   antLocale?: { [key: string]: any };
 }
 
+export interface ComponentClass<P> {
+  [key: string]: any;
```

Comment:
If we don't extends from `React.ComponentClass<P>` will lose the default static members.